mlflow/sklearn/__init__.py

```diff
@@ -465,6 +466,16 @@ def load_model(model_uri):
     )
 
 
+_api_requiring_disabling_autolog_list = [
+    "cross_validate",
+    "cross_val_predict",
+    "cross_val_score",
+    "learning_curve",
+    "permutation_test_score",
+    "validation_curve",
+]
```

Comment:
@dbczumar How would you name this variable?

Reply:
I renamed to `_sklearn_api_autolog_blocklist`

Reply:
@WeichenXu123 What about `_apis_autologging_disabled`?